decorator.rb

```diff
@@ -0,0 +1,22 @@
+require './nameable'
+
+class Decorator < Nameable
+  attr_accessor :person
+
+  def initialize(person)
+    super()
+    @person = person
+  end
```

Comment:
- [ ] As required by the previous reviewer, you are missing a method `correct_name` for the `Decorator` class. Kindly implement that to have your project approved.
```
    def correct_name
      @nameable.correct_name
    end
```
and do not forget to have the `attr_accessor` for the `nameable` variable.

Reply:
Done
![inputourput](https://user-images.githubusercontent.com/13853450/187024468-89636e05-dd93-475e-a861-845344d099d1.png)